modules/tinymce/src/core/test/ts/browser/EditorUploadTest.ts

```diff
@@ -207,7 +207,7 @@ UnitTest.asynctest('browser.tinymce.core.EditorUploadTest', (success, failure) =
     editor.uploadImages((result) => {
       assertResult(editor, uploadedBlobInfo, result);
 
-      return editor.uploadImages((result) => {
+      editor.uploadImages((result) => {
```

Comment:
This seems odd that it wants to remove the the return here? I guess it's because we don't do anything with the parent promise?

Reply:
It was complaining about the function to upload images returning a promise when it was meant to be void.